scripts/install-glab.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Platform check
+PLATFORM=`uname`_`uname -m`
+
+PWD=`pwd`
+
+echo $PWD
+
+# Fetch the appropriate OS binary for glab
+wget -O bin/glab_1.22.0_$PLATFORM.tar.gz  https://github.com/profclems/glab/releases/download/v1.22.0/glab_1.22.0_$PLATFORM.tar.gz
```

Comment:
This won't work for MacOS..
`$PLATFORM` comes out as `Darwin` for MacOS, but the url for tar file is in the below format

`https://github.com/profclems/glab/releases/download/v1.22.0/glab_1.22.0_macOS_x86_64.tar.gz`